lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -2015,13 +2015,17 @@ int prArrayLace(struct VMGlobals *g, int numArgsPushed)
 	numLists = obj1->size;
 
 	if(IsNil(b)) {
+		// If length argument is unspecified, compute length as the multiple of
+		// the length of the shortest sub-array and the number of sub-arrays.
```

Comment:
shouldn't it read `the longest sub-array`?

Reply:
Nope, the comment describes the code accurately as written. Running on stock sc:

```supercollider
[[1, 2, 3, 4], [5, 6, 7], [8]].lace();  // --> [1, 5, 8]
```

Of course, I could see an argument for either minimum or maximum length, it is an unusual case when there are both nonarray items in the list and an unspecified length, but changing to maximum length will represent a change from the current behavior.

Reply:
I see, I wouldn't have expected this. But in this case, the length is not the shortest sub-array (that would be `1`), but the size of the array itself.

Somehow, I'd prefer to have the non-trivial default of a full unroll, instead of having to write
`array.lace(array.maxItem(_.size).size) * array.size`

But I do see that we don't want to change behavior here.